### PR TITLE
Add artifacts and sprint 1 blueprint

### PR DESCRIPTION
**Pull request by diegoro1**
A starting point for our sprint 1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,22 @@ For internet users who find themselves having a difficult time organizing and di
 - Jerrett Longworth (@jcoolrace)
 - Diego Rodrigues (@diegoro1)
 - Jaeivan Romero (@jaerom)
+
+# Sprint 1
+
+- [Product Backlog](https://trello.com/b/ui7fei2w/clippy)
+- [Sprint Backlog](https://trello.com/b/ui7fei2w/clippy)
```
Comment (jcoolrace) [inline README.md:18]: Want to also add the requirements in here while we're at it?

```suggestion
- [Sprint Backlog](https://trello.com/b/ui7fei2w/clippy)
- [Requirements](artifacts/requirements.md)
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Clippy
 
-## Product Vision
+# Product Vision
```
Comment (jcoolrace) [inline README.md:3]: Sorry, I had to do it :P

```suggestion
## Product Vision
```

**Review by jcoolrace — CHANGES_REQUESTED** [review]
Blueprint looks good. Got a few small tweaks for 'ya.

File: README.md [hunk 3]
```diff
@@ -11,3 +11,22 @@ For internet users who find themselves having a difficult time organizing and di
 - Jerrett Longworth (@jcoolrace)
 - Diego Rodrigues (@diegoro1)
 - Jaeivan Romero (@jaerom)
+
+# Sprint 1
```
Comment (jcoolrace) [inline README.md:15]: Same thing here :P

```suggestion
# Sprint 1
## Sprint 1
```

File: README.md (hunk 2 above)
Comment (jcoolrace) [reply README.md:3]: On closer look, this is fine. The original markdown header hierarchy makes more sense.

**Review by jcoolrace — APPROVED** [review]
On closer look, the original markdown hierarchy makes more sense.

**Comment by diegoro1** [comment]
> On closer look, the original markdown hierarchy makes more sense.

I was making the same changes at first, I changed my mind after looking at the doc as a whole haha!

**Comment by jordynhayden** [comment]
Looks good.

**Review by jordynhayden — APPROVED** [review]

**Pull request merged**